test/templates/nstemplatetiers/appstudio/spacerole_admin.yaml

```diff
@@ -61,7 +61,9 @@ objects:
       - gitopsdeploymentrepositorycredentials
       - gitopsdeploymentsyncruns
     verbs:
-    - '*'
+      - get
```

Comment:
You don't really have to update the test yaml. It's used to test that the templates are loaded. The content doesn't matter.
It doesn't hurt to upload it to but it's not necessary. Just FYI for future PRs :)